public/locales/ru-RU/translations.json

```diff
@@ -610,5 +610,13 @@
     "removeBtn": "Удалить",
     "createBtn": "Создать",
     "startBtn": "Начать"
+  },
+  "orderContexts": {
+    "exchange": "Обменный",
+    "margin": "Маржинальный",
+    "funding": "Дериватив"
+  },
+  "orderStatuses": {
+    "ACTIVE": "АКТИВНЫЙ"
```

Comment:
do not include non-english translations to PR as we have workflow for translations:
1. initially in PR only add english strings
2. ask to translate them in all other supported languages using crowdin project where dedicated team members will translate for respective language
3. once tasks are completed in crowdin, download other language strings and commit in separate PR